.infrastructure/deployment.yml

```diff
@@ -45,8 +45,27 @@ spec:
         - name: SECRET_KEY
           valueFrom:
             secretKeyRef:
-              name: app-secret
+              name: mysql-secrets
```

Comment:
The secret name `mysql-secrets` should be corrected to `mysql-secret` to match the actual secret name used in your configuration files.